staging/src/k8s.io/apiserver/pkg/util/webhook/client.go

```diff
@@ -222,6 +222,7 @@ func (cm *ClientManager) HookClient(cc ClientConfig) (*rest.RESTClient, error) {
 	cfg := rest.CopyConfig(restConfig)
 	cfg.Host = u.Scheme + "://" + u.Host
 	cfg.APIPath = u.Path
+	cfg.TLSClientConfig.Insecure = true
```

Comment:
Need to remove, this was to just help me testing a webhook via URL without having to spend too much time on certs.